docs/installing-metabase.md

```diff
@@ -0,0 +1,16 @@
+# Installing Metabase
+
+Welcome to Metabase! We’re excited to help you get started. Metabase has been designed around the idea that getting access to your data should be pain-free - and that starts with our install process.
```

Comment:
`Metabase has been designed around the idea that getting access to your data should be pain-free - and that starts with our install process.` while accurate, reads a bit stiff. If it were me, I'd go with something like `We've made our install process as simple as we can manage to get you right to your data.`